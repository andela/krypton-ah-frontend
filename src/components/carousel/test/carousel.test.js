import React from 'react';
import { shallow } from 'enzyme';
import Carousel from '..';

describe('home container test', () => {
  it('should test for the home component', () => {
    const wrapper = shallow(<Carousel />);
    expect(wrapper.find('Grid')).toBeDefined();
  });
});
