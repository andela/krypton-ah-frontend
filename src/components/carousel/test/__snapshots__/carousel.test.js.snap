// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`carousel conatiner test should test for the carousel component 1`] = `
<GridColumn>
  <Carousel
    autoPlay={true}
    axis="horizontal"
    centerMode={false}
    centerSlidePercentage={80}
    dynamicHeight={false}
    emulateTouch={false}
    infiniteLoop={true}
    interval={3000}
    onChange={[Function]}
    onClickItem={[Function]}
    onClickThumb={[Function]}
    selectedItem={0}
    showArrows={true}
    showIndicators={true}
    showStatus={false}
    showThumbs={false}
    statusFormatter={[Function]}
    stopOnHover={true}
    swipeScrollTolerance={5}
    swipeable={true}
    transitionTime={350}
    useKeyboardArrows={false}
    verticalSwipe="standard"
    width="100%"
  >
    <div
      className="carousel-container"
      key="65719288-0395-445e-b587-2b98b70bdec9"
    >
      <Image
        as="img"
        size="massive"
        src="https://farm4.staticflickr.com/3894/15008518202_c265dfa55f_h.jpg"
        ui={true}
      />
      <p
        className="legend text"
      >
        <Link
          replace={false}
          to="/article/65719288-0395-445e-b587-2b98b70bdec9"
        >
          <p
            className="title"
          >
            This is a post title
          </p>
        </Link>
        <p
          className="author"
        >
          hakeem ibrahim
        </p>
        <Connect(Icons)
          date="2019-01-23T07:09:10.447Z"
        />
      </p>
    </div>
  </Carousel>
</GridColumn>
`;

exports[`carousel conatiner test should test for the carousel component 2`] = `
<GridColumn>
  <Carousel
    autoPlay={true}
    axis="horizontal"
    centerMode={false}
    centerSlidePercentage={80}
    dynamicHeight={false}
    emulateTouch={false}
    infiniteLoop={true}
    interval={3000}
    onChange={[Function]}
    onClickItem={[Function]}
    onClickThumb={[Function]}
    selectedItem={0}
    showArrows={true}
    showIndicators={true}
    showStatus={false}
    showThumbs={false}
    statusFormatter={[Function]}
    stopOnHover={true}
    swipeScrollTolerance={5}
    swipeable={true}
    transitionTime={350}
    useKeyboardArrows={false}
    verticalSwipe="standard"
    width="100%"
  >
    <div
      className="carousel-container"
      key="65719288-0395-445e-b587-2b98b70bdec9"
    >
      <Image
        as="img"
        size="massive"
        src="https://farm4.staticflickr.com/3894/15008518202_c265dfa55f_h.jpg"
        ui={true}
      />
      <p
        className="legend text"
      >
        <Link
          replace={false}
          to="/article/65719288-0395-445e-b587-2b98b70bdec9"
        >
          <p
            className="title"
          >
            This is a post title
          </p>
        </Link>
        <p
          className="author"
        >
          hakeem ibrahim
        </p>
        <Connect(Icons)
          date="2019-01-23T07:09:10.447Z"
        />
      </p>
    </div>
  </Carousel>
</GridColumn>
`;
